### ISSUE TITLE
Allow to define the name of the assets

### ISSUE THREAD

**Issue by piranna** [remixz/publish-release#2]
Instead of only allow a list of strings, allow an object where keys are the assets and the value the name they will have when uploaded. As a plus, allow a list of objects too, so both objects and strings (when the name will be the same) can be combined.

**Comment by remixz** [comment]
good idea! for backwards compatibility, it would always take in array, but then it could look like this:

```js
var assets = [ '/path/to/file.txt', { name: 'foo.txt', path: '/path/to/bar.txt' } ]
```

**Comment by piranna** [comment]
What you say it's just what I added in the "as a plus" section ;-) Good to know you liked it! :-D
